pyproject.toml

```
@@ -7,3 +7,9 @@ requires = [
 [tool.autopep8]
 max_line_length = 120
 ignore = ["W504", "W504", "E402", "E731", "C40", "E741", "F40", "F841"]
+
+[tool.black]
+# https://github.com/psf/black
+line-length = 88
```

This project is operating with a line length of 120 chars, so this line needs to be changed.

```suggestion
line-length = 120
```